backend/src/main/java/de/echochamber/backend/controller/PollController.java

```diff
@@ -130,7 +138,7 @@ private PollEntity map(Poll poll) {
 
         for (int i = 0; i < poll.getPossibleAnswers().size(); i++) {
```

Comment:
du könntest hier die gleiche Kurzschreibweise wie in der for-Schliefe oben verwenden, dann ist es einheitlich